docs/en/Configuring-Alluxio-with-HDFS.md

```diff
@@ -33,8 +33,7 @@ $ mvn install -P<YOUR_HADOOP_PROFILE> -DskipTests
 
 Alluxio provides predefined build profiles including `hadoop-1`, `hadoop-2.2`, `hadoop-2.3` ...
 `hadoop-2.8` for different distributions of Hadoop. If you want to build Alluxio with a specific
-Hadoop release version, you can also specify the version `<YOUR_HADOOP_VERSION>` in the command.
-For example,
+Hadoop release version, you can also specify the version in the command. For example,
```

Comment:
when using HDFS as a UFS, should users set `ufs.hadoop.version` instead of the hadoop profile?

Reply:
it is essentially the same, as 
`<ufs.hadoop.version>${hadoop.version}</ufs.hadoop.version>` in hdfs module pom.
I view `ufs.hadoop.version` as an internal property, that we should not document to complicate the tutorial, whereas `hadoop.version` still works and is backward compatible.

Reply:
got it, thanks for explaining